### ISSUE TITLE
Design web page for Press Office to replace a PDF file

### ISSUE THREAD

**Issue by dorothyyeager** [fecgov/fec-cms#4365]
### Summary

**What we are after:**
_The Press Office maintains a list of the largest civil penalties in FEC cases. As a short-term fix to get it off transition, they are making a quick PDF. But they'd really like a web page for it, so this ticket is to design that._

### Related issues

_List any relevant related issue(s)_

- #4144 - Old press files on transition needing redirect

### Completion criteria

- [ ] New page is designed
- [ ] Implementation ticket created.

### Tech steps or considerations (optional)

_List any considerations the tech team should know. Additionally, any specific tech steps can be included here._

- [ ] First
- [ ] Next
- [ ] Then

### Future work

_Other future work that may be needed following this issue's completion._

**Comment by djgarr** [comment]
Moving to 14.6 as we're waiting on Press for additional content

**Comment by johnnyporkchops** [comment]
Closing in favor of larger goal for Press pages.
